ui/packages/catalog-ui-search/src/main/webapp/extension-points/table-export/table-export.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import { useEffect, useState } from 'react'
+import TableExport from '../../react-component/table-export'
+import {
+  getExportOptions,
+  Transformer,
+} from '../../react-component/utils/export'
+
+type Option = {
+  label: string
+  value: string
+}
+
+type ExportResponse = {
+  displayName: string
+  id: string
+}
+
+export type Props = {
+  selectionInterface: any
+  exportFormats: Option[]
+}
+
+const TableExports = (props: Props) => {
+  const [formats, setFormats] = useState([])
+
+  useEffect(() => {
+    const fetchFormats = async () => {
+      const response = await getExportOptions(Transformer.Query)
+      const exportFormats = await response.json()
```

Comment:
I don't know who else is using `getExportOptions` but it would be better if it returned the JSON directory to you instead of having you get the json from it.

Reply:
`getExportOptions` is also used in `export-interaction` and `query-interaction` in which the response, not the json, is passed directly into `ResultsExport` as a prop

Reply:
I think I can do that